roles/openshift_health_checker/openshift_checks/mixins.py

```diff
@@ -2,17 +2,14 @@
 from openshift_checks import get_var
 
 
-class NotContainerized(object):
+class NotContainerizedMixIn(object):
```

Comment:
I read the first class declaration as "**Package Availability is a Not Containerized OpenShift Check**".
It could have been "My Custom Check is a Enterprise Only Not Containerized OpenShift Check":

```python
class MyCustomCheck(EnterpriseOnly, NotContainerized, OpenShiftCheck):
```


Adding "Mixin" suffixes is just noise.

"MixIn" with camel case is even more strange, the term is a single word AFAIK: https://en.wikipedia.org/wiki/Mixin

---

In Python, perhaps expect for exceptions ;), we don't normally prefix or suffix names with some part of its "type". We don't say FooClass, or BarAbstractInterface, or IReader (I think this might be the convention in other languages like Java, C# or whatnot).